packages/form-core/src/FormApi.ts

```diff
@@ -170,6 +168,18 @@ export class FormApi<TFormData, ValidatorType> {
     this.update(opts || {})
   }
 
+  mount = () => {
+    if (typeof this.options.onMount === 'function') {
+      return this.options.onMount(this.state.values, this)
+    }
+    if (this.options.validator) {
+      return (this.options.validator as Validator<TFormData>)().validate(
+        this.state.values,
+        this.options.onMount,
+      )
+    }
```

Comment:
Let's remove `validator`. It's mostly a holdover from the older API we had before `onMount`/etc.

Reply:
Do you mean remove the `validator` option entirely from the form or do you mean we should not run the validator in the `mount` function? If we removed the validator option from the Form then you would have to manually specify the validator in every `Field`.

Reply:
OH! You're gonna laugh - I legit forgot what `validator` was 🤦‍♂️

Disregard!